spec.html

```diff
@@ -24990,10 +24994,14 @@ <h1>CreateDynamicFunction ( _constructor_, _newTarget_, _kind_, _args_ )</h1>
             1. Let _prefix_ be the prefix associated with _kind_ in <emu-xref href="#table-dynamic-function-sourcetext-prefixes"></emu-xref>.
             1. Let _sourceString_ be the string-concatenation of _prefix_, *" anonymous("*, _P_, 0x000A (LINE FEED), *") {"*, _bodyString_, and *"}"*.
             1. Let _sourceText_ be ! StringToCodePoints(_sourceString_).
+            1. Let _parameters_ be ParseText(! StringToCodePoints(_P_), _parameterSym_).
+            1. If _parameters_ is a List of errors, throw a *SyntaxError* exception.
+            1. Let _body_ be ParseText(! StringToCodePoints(_bodyString_), _bodySym_).
+            1. If _body_ is a List of errors, throw a *SyntaxError* exception.
+            1. NOTE: The parameters and body are parsed separately to ensure that each is valid alone. For example, `new Function("/*", "*/ ) {")` is not legal.
```

Comment:
This note doesn't need to be inline in the algorithm steps. Probably better as an `<emu-note>`.

Reply:
Eh, I mildly prefer it here, but I'm willing to move it if you feel strongly or @syg agrees.